ingestion/src/metadata/ingestion/ometa/mixins/glossary_mixin.py

```diff
@@ -13,19 +13,32 @@
 
 To be used be OpenMetadata
 """
-from typing import TypeVar
+import json
+import traceback
+from typing import Optional, Type, TypeVar, Union
 
 from pydantic import BaseModel
 
 from metadata.generated.schema.entity.data.glossary import Glossary
 from metadata.generated.schema.entity.data.glossaryTerm import GlossaryTerm
+from metadata.generated.schema.type import basic
+from metadata.ingestion.ometa.utils import model_str
 from metadata.utils.logger import ometa_logger
 
-T = TypeVar("T", bound=BaseModel)
+from .patch_mixin_utils import (
```

Comment:
can we have imports with the full path here?

Reply:
Fixed